src/main/java/seedu/address/logic/parser/AddCommandParser.java

```diff
@@ -28,11 +28,17 @@ public class AddCommandParser implements Parser<AddCommand> {
      * @throws ParseException if the user input does not conform the expected format
      */
     public AddCommand parse(String args) throws ParseException {
+        String getName = (args.split("-", 2))[0];
```

Comment:
This is a different level of abstraction compared to the lines after that. Consider putting this in another method.